openfecwebapp/api_caller.py

```diff
@@ -148,7 +148,25 @@ def landing_mock_data():
 def load_top_candidates(sort):
         response = _call_api(
             'candidates', 'totals',
-            sort_hide_null=True, election_year=2016, cycle=2016, election_full=True, sort=sort, per_page=5
+            sort_hide_null=True, election_year=2016, cycle=2016, election_full=False, sort=sort, per_page=5
+        )
+        if response['results']:
+            return response['results']
+        return {}
+
+def load_top_pacs(sort):
+        response = _call_api(
+            'totals', 'pac',
+            sort_hide_null=True, cycle=2016, sort=sort, per_page=5
+        )
+        if response['results']:
```

Comment:
Right, at least that's my understanding looking at most of the code in this file (in some cases, the `results` are just a list, not a dictionary): if nothing is there, return an empty object of some sort so the page will still render.

I agree that making a separate PR for this makes sense given that multiple methods could be improved and that a proper error state should be designed to handle that scenario.